tasks/install_shell.sh

```diff
@@ -35,6 +35,25 @@ exists() {
   fi
 }
 
+# Check whether the apt config file has been modified, warning and exiting early if it has
+assert_unmodified_apt_config() {
+  list_file=/etc/apt/sources.list.d/puppet.list
```

Comment:
With the latest puppet docs this would actually be puppet6.list - we should definitely look for both. I don't think dpkg will let a user have both at the same time, so just checking for whichever exists is sufficient until someone runs into issues with both existing.